src/encoded/static/components/award.js

```diff
@@ -2045,11 +2025,6 @@ class CumulativeGraph extends React.Component {
                 data: {
                     labels: monthReleased,
                     datasets: [{
```

Comment:
I'm not very familiar with chartjs, but is there a simpler way to write this if there's only one dataset? Or is it better to keep it this way since it's possible that we'll want to add back submitted data at some point?

Reply:
I’m not sure, but I’m not seeing anything significantly simpler in their samples, e.g. https://github.com/chartjs/Chart.js/blob/master/samples/charts/line/point-styles.html#L36